control/control_loop.go

```diff
@@ -330,17 +330,29 @@ func (l *Loop) GetConfig(ctx context.Context) Config {
 func (l *Loop) MonitorTuning(ctx context.Context) {
 	// wait until tuning has started
 	for {
-		tuning := l.GetTuning(ctx)
-		if tuning {
-			break
+		// 100 Hz is probably faster than we need, but we needed at least a small delay because
+		// GetTuning will lock the PID block
+		if utils.SelectContextOrWait(ctx, 10*time.Millisecond) {
```

Comment:
I can try to remove these if ya want. Was worried that calling GetTuning was affecting things at the time when I added this.

Reply:
this is fine and probably safe. happy with leaving it.